Backend/app/dashboard/rider-dashboard/src/API/RiderPlatform/RideBooking/Maps.hs

```diff
@@ -0,0 +1,79 @@
+{-
+ Copyright 2022-23, Juspay India Pvt Ltd
+
+ This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License
+
+ as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program
+
+ is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+
+ or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of
+
+ the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+
+module API.RiderPlatform.RideBooking.Maps where
+
+import qualified "rider-app" API.Dashboard.RideBooking.Maps as BAP
+import qualified Dashboard.RiderPlatform.Customer as Common
+import qualified "rider-app" Domain.Action.UI.Maps as DMaps
+import qualified "lib-dashboard" Domain.Types.Merchant as DM
+import qualified "rider-app" Domain.Types.Person as DP
+import qualified Domain.Types.Transaction as DT
+import "lib-dashboard" Environment
+import Kernel.Prelude
+import Kernel.Types.Id
+import Kernel.Utils.Common
+import qualified RiderPlatformClient.RiderApp as Client
+import Servant
+import qualified SharedLogic.Transaction as T
+import "lib-dashboard" Tools.Auth
+import Tools.Auth.Merchant
+
+type API =
+  "maps"
+    :> ( ApiAuth 'APP_BACKEND 'WRITE_ACCESS 'CUSTOMERS
+           :> BAP.RideAutoCompleteAPI
+           :<|> ApiAuth 'APP_BACKEND 'WRITE_ACCESS 'CUSTOMERS
+             :> BAP.RideGetPlaceDetailsAPI
+           :<|> ApiAuth 'APP_BACKEND 'READ_ACCESS 'CUSTOMERS
+             :> BAP.RideGetPlaceNameAPI
```

Comment:
Why some APIs need `READ` while some need `WRITE` access?

Reply:
@0utkarsh Get APIs only need read access and for POST APIs i have given write access

Reply:
All these are POST, right?

Reply:
ohh got it, will change this